exercises/I-array-properties/exercise.js

```diff
@@ -6,7 +6,7 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
+  return arr.length===0; // complete this statement
```

Comment:
Yep, this is how I would do it too. An alternative is to write `!arr.length` - can you understand why that would also work? NOTE: I'm not suggesting that the alternative is better - it isn't. This is just to get you thinking :)